roles/pulp_api/templates/pulpcore-api.service.j2

```diff
@@ -14,9 +14,13 @@ User={{ pulp_user }}
 Group={{ pulp_group }}
 PIDFile=/run/pulpcore-api.pid
 RuntimeDirectory=pulpcore-api
+
+# timeout 90 is needed Pulp to service its 1st request on extremely slow
+# machines, such as qemu-emulated 2-core machines
```

Comment:
I think it needs a new changelog entry

Reply:
Yeah, I agree. I'll make it a separate commit.

Reply:
@gerrod3 this is the change I mentioned